src/linked_list.js

```diff
@@ -0,0 +1,172 @@
+'use strict'
+
+class Node {
+  constructor(data, next) {
+    this.data = data
+    this.next = next
+  }
+}
+
+export default class LinkedList {
+  constructor() {
+    this.list = null
+    this.size = 0
+  }
+
+  getHeadNode() {
+    return this.list
+  }
+
+  getTailNode() {
+    let temp = this.list
+    while (temp.next) {
+      temp = temp.next
+    }
+    return temp
+  }
+
+  contains(value) {
+    let status = false
```

Comment:
@jhallman5 So in this case, instead of setting status to true in the while loop, I could just `return true` and default to `return false`?